vendor/k8s.io/system-validators/validators/docker_validator.go

```diff
@@ -32,11 +33,13 @@ type DockerValidator struct {
 }
 
 // dockerInfo holds a local subset of the Info struct from
-// github.com/docker/docker/api/types.
+// https://github.com/docker/cli/blob/master/cli/command/system/info.go
+// and https://github.com/moby/moby/blob/master/api/types/types.go
 // The JSON output from 'docker info' should map to this struct.
 type dockerInfo struct {
-	Driver        string `json:"Driver"`
-	ServerVersion string `json:"ServerVersion"`
+	Driver        string   `json:"Driver"`
+	ServerVersion string   `json:"ServerVersion"`
+	ServerErrors  []string `json:",omitempty"`
```

Comment:
This is odd. How does this deserialize?

Reply:
it's only deserialized for the time being:
```
{ "ServerErrors":["some message"] }

 v

dockerInfo{ServerErrors: []string{"some message"} }
```

was added here:
https://github.com/kubernetes/system-validators/pull/3